src/store/getters.ts

```diff
@@ -67,8 +68,8 @@ export const isMobile = (state: StateIF): boolean => {
 }
 
 /** True if user is authenticated, else False. */
-export const isAuthenticated = (): boolean => {
-  return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
+export const isAuthenticated = (rootState: RootStateIF): boolean => {
+  return Boolean(rootState.auth && rootState.auth.token)
```

Comment:
You can clean this up a bit with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining :)

Reply:
Fixed, thanks.